datadog.conf.example

```diff
@@ -88,6 +93,11 @@ use_mount: no
 # histogram_aggregates: max, median, avg, count
 # histogram_percentiles: 0.95
 
+# Multiple endpoints/api_keys
```

Comment:
Indeed, it's just that our customers shouldn't be using it. But you're right, it should be specified anyway. I added it. ✅